Microsoft.Toolkit.Uwp.SampleApp.Shared/Microsoft.Toolkit.Uwp.SampleApp.Shared.projitems

```diff
@@ -464,6 +464,8 @@
     <Content Include="$(MSBuildThisFileDirectory)Assets\UWPCommunityToolkitSampleAppAppList.scale-150.png" />
     <Content Include="$(MSBuildThisFileDirectory)Assets\UWPCommunityToolkitSampleAppAppList.scale-200.png" />
     <Content Include="$(MSBuildThisFileDirectory)Assets\UWPCommunityToolkitSampleAppAppList.scale-400.png" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != 'MonoAndroid'">
```

Comment:
Add a comment here associating this with https://github.com/unoplatform/uno/issues/1370